webpack/scenes/ContentViews/ContentViewsActions.js

```diff
@@ -15,4 +22,25 @@ const getContentViews = extraParams => get({
   params: createContentViewsParams(extraParams),
 });
 
+const cvCreateSuccess = (response, dispatch) => {
+  const { data: { name } } = response;
+  return dispatch(addToast({
+    type: 'success',
+    message: __(' Content view ' + name + ' created'),
+  }));
+};
```

Comment:
could the `successToast` param work for the notification? this also helps us not pass in the dispatch below

https://github.com/theforeman/foreman/blob/a58aa42c9b5aa39ea2209dd01a06334ddfbd777d/webpack/assets/javascripts/react_app/redux/API/APIRequest.js#L20

some of the other actions didn't use this as it wasn't a feature in the API middleware at the time (we should update this at some point)

Reply:
same question error handling for using `errorToast`

Reply:
Updated..